plugins/obs-remote

```diff
@@ -0,0 +1 @@
+../../obs-remote/
```

Comment:
This file seems to not be a currently included core/first-party plugin?  Was this added by mistake?